acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -433,24 +446,61 @@ def _create_model_cos_zenith_binned(self,
             A dict with observation number as key and a background model that could be used as an acceptance model associated at each key
 
         """
+        # Determine binning method
+        methods = np.array(['min_livetime', 'min_livetime_per_wobble', 'min_n_observation', 'min_n_observation_per_wobble'])
+        if self.cos_zenith_binning_method in methods: i_method = np.where(methods == self.cos_zenith_binning_method)[0][0]
+        else: logging.error(f"{self.cos_zenith_binning_method} not in cos zenith binning methods")
+
+        per_wobble = i_method in [1,3]
```

Comment:
Proposal code : 

```
methods = {'min_livetime':1, 'min_livetime_per_wobble':-1, 'min_n_observation':2, 'min_n_observation_per_wobble':-2}
try:
    i_method  = methods[self.cos_zenith_binning_method]
except KeyError:
    logging.error(f"{self.cos_zenith_binning_method} not a valid zenith binning method")
per_wobble = i_method<0

```

Why : 

1. Probably faster to access a dict than to check an array with "in", and then running np.where
2. Easier to add per_wobble methods if the boolean code doesn't change, probably needs a comment that negative values are associated to "per_wobble" methods.

Reply:
Done